ibis_substrait/tests/compiler/snapshots/test_tpch/test_compile/tpc_h092/tpc_h092.json

```diff
@@ -924,25 +924,13 @@
                                   "arguments": [
                                     {
                                       "value": {
-                                        "cast": {
-                                          "type": {
-                                            "decimal": {
-                                              "scale": 2,
-                                              "precision": 15,
-                                              "nullability": "NULLABILITY_NULLABLE"
-                                            }
-                                          },
-                                          "input": {
-                                            "selection": {
-                                              "directReference": {
-                                                "structField": {
-                                                  "field": 26
-                                                }
-                                              },
-                                              "rootReference": {}
+                                        "selection": {
+                                          "directReference": {
+                                            "structField": {
+                                              "field": 26
                                             }
                                           },
-                                          "failureBehavior": "FAILURE_BEHAVIOR_THROW_EXCEPTION"
+                                          "rootReference": {}
```

Comment:
This is the only diff relative to `tpc_h090.json`:

<img width="1512" alt="Screenshot 2023-12-30 at 16 11 10" src="https://github.com/ibis-project/ibis-substrait/assets/7870972/4b03e8bd-a135-4292-8e27-7aa36c9146f5">